composer.json

```diff
@@ -67,6 +68,7 @@
         "omnipay/2checkout": "~2.0",
         "omnipay/authorizenet": "~2.0",
         "omnipay/buckaroo": "~2.0",
+        "cardgate/omnipay-cardgate": "~2.0",
```

Comment:
That's a good point. We should probably add a new "recommends" section for the third party gateways.